core/src/components/textarea/textarea.scss

```diff
@@ -71,8 +73,6 @@
   flex: 1;
 
   background: var(--background);
-
```

Comment:
Why did this move?

Reply:
This was just clean-up, aligning closer to the original property ordering:
- https://github.com/ionic-team/ionic-framework/blob/feature-7.0/core/src/components/textarea/textarea.scss#L48 (feature-7.0)
- https://github.com/ionic-team/ionic-framework/blob/main/core/src/components/textarea/textarea.scss#L48 (main)